udata/tests/api/test_datasets_api.py

```diff
@@ -607,20 +606,18 @@ def test_list(self):
 
     def test_create(self):
         data = self.factory.as_dict()
-        with self.api_user():
```

Comment:
`self.api_user()` logic seems useless here as the setup, played before every test logs in an `AdminFactory()`